.gitignore

```diff
@@ -5,4 +5,5 @@ src
 env.js
 nginx.conf
 data/audit.json
-admin.json
\ No newline at end of file
+admin.json
+keys/
```

Comment:
While not entirely necessary, it is helpful to prevent leaking the public key in any circumstance.